website/content/ja/contribute/projects/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: プロジェクト
+list_pages: true
+menu:
+  main:
+    weight: 20
+description: これは参加する機会のリストです
+---
+
+TAGプロジェクトは[GitHub Issues](https://github.com/cncf/tag-app-delivery/issues)で管理しています。現在あるプロジェクトや新しいプロジェクトを提案したい場合はこちらから始めてください。
```

Comment:
When translating sentences such as `... start there for [noun] and to [verb]` directly, it may sound awkward in Japanese. How about adding a verb at the beginning to improve the flow?

```suggestion
TAGプロジェクトは[GitHub Issues](https://github.com/cncf/tag-app-delivery/issues)で管理しています。進行中のプロジェクトを確認したり、新しいプロジェクトを提案したい場合は、こちらから始めてください。
```